stl/inc/vector

```diff
@@ -1039,6 +1039,12 @@ public:
         pointer& _Myfirst = _My_data._Myfirst;
         pointer& _Mylast  = _My_data._Mylast;
 
+#if _ITERATOR_DEBUG_LEVEL == 2
+        const_pointer _Valptr = addressof(_Val);

```

Comment:
`addressof` needs to be `_STD` qualified in order to avoid our nemesis, Argument-Dependent Lookup.